pkg/metrics/exporter.go

```diff
@@ -60,8 +60,12 @@ func GenerateMetrics(state health.State) string {
 		metricCheckState[metricName] = checkStatus
 		runDuration, err := time.ParseDuration(d.RunDuration)
 		if err != nil {
-			log.Errorln("Error parsing run duration:", d.RunDuration, "for metric:", metricName, "error:", err)
-			continue
+			if d.RunDuration != "" {
```

Comment:
do we need this `if` here?  It seems like if there was an error parsing the duration, the `time.ParseDuration` call on 61 would have found it.  At this point the only possible "bad" duration that `runDuration` could be set to would be `0`.  you could just check that, but check it after this `err != nil` if statement.

Reply:
Same with the code down on line 90